tsl/org.testeditor.tsl.dsl.web/webapp/xtext-resources/generated/mode-tsl.js

```diff
@@ -1,39 +1,14 @@
 define(["ace/lib/oop", "ace/mode/text", "ace/mode/text_highlight_rules"], function(oop, mText, mTextHighlightRules) {
 	var HighlightRules = function() {
-    // available tokens: tag, type, keyword, text, string, selection, comment, function, constant.buildin, constant.library, comment, comment.doc, comment.doc.tag, xml-pe, meta.tag, entity.other.attribute-name
```

Comment:
This file has no Windows line-feeds... it's generated in the MWE 2 workflow - I think we're missing a property for the line deliminter.